libraries/plugins/rc/resource_count.cpp

```diff
@@ -3,35 +3,238 @@
 
 #include <steem/protocol/transaction.hpp>
 
+#define STATE_BYTES_SCALE              10000
+#define STATE_TRANSACTION_BYTE_SIZE      175
+
 namespace steem { namespace plugins { namespace rc {
 
 using namespace steem::protocol;
 
+struct state_object_size_info
```

Comment:
The whitespace in this struct is kind of all over the place.

I am all for adding whitespace to align constants. Not sure whats up with `*STATE_BYTES_SCALE`. I don't think we've ever had multiplication without whitespace. Looks like a pointer to me.